evap/student/tests/test_views.py

```diff
@@ -26,10 +26,10 @@ class TestVoteView(WebTest):
 
     @classmethod
     def setUpTestData(cls):
-        cls.voting_user1 = baker.make(UserProfile)
-        cls.voting_user2 = baker.make(UserProfile)
-        cls.contributor1 = baker.make(UserProfile)
-        cls.contributor2 = baker.make(UserProfile)
+        cls.voting_user1 = baker.make(UserProfile, email="voting_user1@institution.example.com")
+        cls.voting_user2 = baker.make(UserProfile, email="voting_user2@institution.example.com")
+        cls.contributor1 = baker.make(UserProfile, email="contributor1@institution.example.com")
+        cls.contributor2 = baker.make(UserProfile, email="contributor2@institution.example.com")
```

Comment:
I think we don't need the email here and can just use `self.voting_user1` as the user argument for `self.app.get` calls below

Reply:
will be dealt with in #1475 (if we find a way to get it working).